src/transformers/generation/utils.py

```diff
@@ -1839,6 +1839,7 @@ def generate(
                 output_scores=generation_config.output_scores,
                 return_dict_in_generate=generation_config.return_dict_in_generate,
                 synced_gpus=synced_gpus,
+                sequential=generation_config.low_memory,
```

Comment:
We should update the documentation regarding `generation_config.low_memory`, to mention that it is also compatible with beam methods :) (see [here](https://github.com/huggingface/transformers/blob/a85ea4b19a47824a8e85d3304a698e2d5c8325ec/src/transformers/generation/configuration_utils.py#L195))